check50.py

```diff
@@ -32,8 +33,8 @@ def main():
     parser.add_argument("files", nargs="*")
     parser.add_argument("-d", "--debug", action="store_true")
     parser.add_argument("-l", "--local", action="store_true")
-    parser.add_argument("-f", "--force", action="store_true")
     parser.add_argument("--log", action="store_true")
+    parser.add_argument("--no-autoupdate", action="store_true")
```

Comment:
Actually, I retract this. For folks who install `check50` on their own machines, it's indeed probably best if updating be the default behavior, else they'll likely never update. I do the same with `cli50` for that reason. For `cli50`, I used `-f`/`--fast`, which I borrowed from, I think, standard GNU arg conventions. (`-f` is easy to type too when I don't want to wait for the Docker image to update.) But since `check50` is server-side, not sure "fast" is the right connotation here. The `--no-something` convention is common. Or we could do `--update=X`, where `X` is documented as `1` by default? Not sure which is the more common convention?